### PR TITLE
Update clause_4_terms_and_definitions.adoc

### PR DESCRIPTION
**Pull request by chris-little**
spatio-temporal data definition

### CODE REVIEW

**Review by m-burgoyne — APPROVED** [review]

**Comment by chris-little** [comment]
Fixes Issue #250

**Review by tomkralidis — CHANGES_REQUESTED** [review]
Perhaps "data associated with a position in space and time" ?

**Review by chris-little — COMMENTED** [review]
Good suggestion. I approve of course

File: candidate-standard/clause_4_terms_and_definitions.adoc [hunk 1]
```diff
@@ -48,6 +48,11 @@ SOURCE: https://www.iso.org/standard/32551.html[ISO 19133:2005]:
 
 region specified with a geographic position and radial distance
 
+[[spatio-temporal-data-definition]]
+=== spatio-temporal data
+
+data associated with a position in space and time
```
Comment (dr-shorthair) [inline candidate-standard/clause_4_terms_and_definitions.adoc:54]: Or 'data associated with position**s** in space and time'

Comment (chris-little) [reply candidate-standard/clause_4_terms_and_definitions.adoc:54]: @dr-shorthair Thank you. We discussed that variant of the definition. Many use cases have multiple data at a single position, which underlies the EDR API. Having multiple pieces of data at multiple positions weakens the direct connection to coordinates, which is the essence of the API. We do not want to preclude data spread across multiple positions, but is the wrong emphasis.
What do you see as the advantage of the 'many to many' definintion?

Comment (chris-little) [reply candidate-standard/clause_4_terms_and_definitions.adoc:54]: EDR API SWG 47 discussed, and agreed `position` preferred to `positions`. But position could be more than a point, such as a polygon, consistent with the ISO position definition.

**Comment by chris-little** [comment]
Agreed at EDR API SWG 47 after discussion

**Pull request closed without merging**